### PR TITLE
Remove @NotNull annotations

### PR DESCRIPTION
**Pull request by geoand**
This are sometimes added by IntelliJ but they should not be part of the Quarkus codebase

### CODE REVIEW

File: integration-tests/gradle/src/main/resources/multi-source-project/src/main/java/org/acme/service/SimpleService.java [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 @ApplicationScoped
 public class SimpleService {
-    @NotNull
```
Comment (gastaldi) [inline integration-tests/gradle/src/main/resources/multi-source-project/src/main/java/org/acme/service/SimpleService.java:9]: Don't forget to remove the import too

**Review by gastaldi — APPROVED** [review]

File: integration-tests/gradle/src/main/resources/multi-source-project/src/main/java/org/acme/service/SimpleService.java (hunk 1 above)
Comment (geoand) [reply integration-tests/gradle/src/main/resources/multi-source-project/src/main/java/org/acme/service/SimpleService.java:9]: I thought Maven already did, but I guess it didn't for these gradle tests

**Comment by famod** [comment]
Couldn't we exclude & ban the jar it's coming from?

**Comment by geoand** [comment]
We should definitely ban them, I haven't looked into that

**Comment by famod** [comment]
It seems to coming from `org/jetbrains/annotations/24.0.1/annotations-24.0.1.jar`.

I think it might be impossible to get rid of it entirely, so that forbidden-apis thingy might be the way to go.

**Pull request merged**